src/main/resources/static/js/MainWindow.js

```diff
@@ -675,448 +761,479 @@ var initial = 1000;
+				exit : "animated fadeOutUp"
+			}
+		});
+		return false;
+	} else {
+
+		$.notify({
+			message : "Import started"
+		}, {
+			type : "info",
+			allow_dismiss : true,
+			placement : {
+				from : "top",
+				align : "left"
+			},
+			animate : {
+				enter : "animated fadeInDown",
+				exit : "animated fadeOutUp"
+			}
+		});
+		addToLog("Import of File " + name + " started.");
+		document.getElementById("progress").value = 0;
+		id = setInterval(progress, initial);
+		$.ajax({
+			type : "POST",
+			url : "importQueue",
+			data : mydata,
+			success : function(e) {
+				addToLog(e);
+				$.notify({
+					message : "Import finished"
+				}, {
+					type : "success",
+					allow_dismiss : true,
+					placement : {
+						from : "top",
+						align : "left"
+					},
+					animate : {
+						enter : "animated fadeInDown",
+						exit : "animated fadeOutUp"
+					}
+				});
+			},
+			error : function(e) {
+				$.notify({
+					message : "Import failed. Check log for errors."
+				}, {
+					type : "danger",
+					allow_dismiss : true,
+					placement : {
+						from : "top",
+						align : "left"
+					},
+					animate : {
+						enter : "animated fadeInDown",
+						exit : "animated fadeOutUp"
+					}
+				});
+				addToLog(e.responseText);
+				clearInterval(id);
+			}
+		});
+	}
 }
 
 function progress() {
-    $.ajax({
-        type: "GET",
-        url: "progress",
-        success: function (response) {
-            var x = false;
-            check(response, x);
-            if (x) {
-                return;
-            }
-            if (response == "Import has not started yet" || response == "File has not been converted yet") {
-                initial = 2 * initial;
-                if (initial >= 16000) {
-                    addToLog("It seems the Import will take quite a while. Stopping progress requests.");
-                    clearInterval(id);
-                    return;
-                }
-                var temp = id;
-                id = setInterval(progress, initial);
-                clearInterval(temp);
-            }
-            if (response != "Finished") {
-                addToLog(response);
-                var resp = response;
-                resp = resp.slice(-3);
-                resp = resp.substring(0, 2);
-                document.getElementById("progress").value = resp;
-            }
-
-        },
-        error: function (e) {
-            if (retry == 5) {
-                addToLog("Progress could not be queried, stopping.");
-                retry = 0;
-                clearInterval(id);
-                return;
-            }
-            retry++;
-            addToLog("Could not get progress. Retrying " + 5 - retry + " more times");
-        }
-    });
+	$
+			.ajax({
+				type : "GET",
+				url : "progress",
+				success : function(response) {
+					var x = false;
+					check(response, x);
+					if (x) {
+						return;
+					}
+					if (response == "Import has not started yet"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)